specification/iotcentral/resource-manager/Microsoft.IoTCentral/preview/2021-11-01-preview/iotcentral.json

```diff
@@ -0,0 +1,1259 @@
+            "schema": {
+              "$ref": "#/definitions/AppListResult"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.IoTCentral/iotApps/{resourceName}/privateEndpointConnections/{privateEndpointName}": {
+      "get": {
+        "tags": [
+          "Networking"
+        ],
+        "description": "Get the metadata of a Private Endpoint Connection for the IoT Central Application.",
+        "operationId": "PrivateEndpoints_Get",
+        "x-ms-examples": {
+          "PrivateEndpoints_Get": {
+            "$ref": "./examples/PrivateEndpoints_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "$ref": "#/parameters/privateEndpointName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The private endpoint connection for the IoT Central Application.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v4/privatelinks.json#/definitions/PrivateEndpointConnection"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "Networking"
+        ],
+        "description": "Update a Private Endpoint Connection.",
+        "operationId": "PrivateEndpoints_Create",
+        "x-ms-examples": {
+          "PrivateEndpoints_Update": {
+            "$ref": "./examples/PrivateEndpoints_Update.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "$ref": "#/parameters/privateEndpointName"
+          },
+          {
+            "name": "PrivateEndpoint",
```

Comment:
We should be careful with the naming convention. For reference, PrivateEndpoint is the Network RP resource. Whereas, the PrivateEndpointConnection is the IoT Central resource. Customers use PrivateEndpoint to reference/communicate with Network RP and PrivateEndpointConnection to reference/communicate with IOT Central RP.

Reply:
good feedback, i didn't know this. i think i found all the relevant references and updated them.